### PR TITLE
Update php-quiz.md

### PR DESCRIPTION
**Pull request by GTekSD**
same question but

### CODE REVIEW

File: php/php-quiz.md [hunk 1]
```diff
@@ -1284,7 +1284,7 @@ while ($fruit_name = current($array)) {
     }
 ```
 
-#### Q98. What does this code return?
+#### Q98. What does this code return? What does this code print?
```
Comment (Ebazhanov) [inline php/php-quiz.md:1287]: please make a duplication, this is not best practice. Later someone could remove `What does this code return?` let's keep both, thanks @GTekSD

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
left comment

File: php/php-quiz.md (hunk 1 above)
Comment (GTekSD) [reply php/php-quiz.md]: #### Q98. What does this code return?
#### Q98. What does this code print?
Does it look okay? whenever someone searching for this question if "return" or "print" will show result not error sound. before I searched "What does this code return?" then its gives not found error, thats why I update for both. Thanks @Ebazhanov

**Review by Ebazhanov — APPROVED** [review]
LGTM 👍

**Pull request merged**